client/components/stylus_lib/index.styl

```diff
@@ -27,6 +27,8 @@ feature-body-width = 680px
 feature-body-w-margin = 720px
 feature-blockquote-width = 940px
 
+news-body-width = 780px
+
```

Comment:
👍 we should also think about adding some of these values to reaction as constants like colors, might make it easier to standardize.